src/Settings/Path.hs

```diff
@@ -146,6 +150,12 @@ packageDbDirectory :: Stage -> FilePath
 packageDbDirectory Stage0 = buildRootPath -/- "stage0/bootstrapping.conf"
 packageDbDirectory _      = "inplace/lib/package.conf.d"
 
+-- | ( During installtion) Path to package database directory of a given 'Stage'.
```

Comment:
> What does "during installation" refer to?

This mean we use `install` to install the packages to somewhere no in `inplace`. This is somewhat a hack itself to fix another hack (note the `inplace` in `ackageDbDirectory _      = "inplace/lib/package.conf.d"`).

I tried to make it a bit more refined, but ending up with the observation that: most paths in `Settings.Path` are relative to `top` and used in such way. So if I plug some absolute path determined by `dest` in some way, it will break the assumption.